Linux.md

```diff
@@ -96,3 +100,18 @@
 
 ### <a name="moveCursorToTheEndOfLine"></a> Moves the cursor to the end of the line
 `Ctrl+e`
+
+### <a name="previousSudoCommand"></a> Runs Previous Sudo command
+`$!!`
+
+### <a name="dnsip"></a> DNS lookup / Find IP of a website
+`dig websitename`
+
+### <a name="loggedInUser"></a>Give information of particular user
+`finger username`
+
+### <a name="memoryMap"></a> Display memory map of disk usage
+`pmap`
+
+### <a name="divideFileColumn"></a> Divide Files into Columns
```

Comment:
not required

Reply:
I have changed the file review it @arverma007

Reply:
Remove this and change the heading for $!!

Reply:
@arverma007 changed the heading check now